docs/6_xccdf_model.md

```diff
@@ -2126,6 +2150,17 @@ Table 11 shows the possible properties of the *\<xccdf:check\>* element.
 |                 |                 |                 | be resolved to  |
 |                 |                 |                 | provide         |
 |                 |                 |                 | content.        |
+|                 |                 |                 |                 |
```

Comment:
I think we should discourage, remove or _at least_ strongly qualify the use of a checksum. Typically, in current use, external check-content is referenced for the express purpose of using content that is updated more frequently than the Benchmark. For example, Benchmark authors often reference a CVE OVAL feed rather than including the OVAL. If a content author were to add a checksum to a CVE feed reference, it would fail as soon as the CVE feed was updated, rendering the Rule uncheckable without ignoring the checksum.

On the other hand, if the rule author is referencing check content that is not updated more frequently than the Benchmark, then there is no need to reference it. It can be included in the Datastream.